targets/TARGET_NXP/TARGET_MCUXpresso_MCUS/TARGET_IMX/sleep.c

```diff
@@ -19,8 +19,7 @@
 
 extern void vPortPRE_SLEEP_PROCESSING(clock_mode_t powermode);
 extern void vPortPOST_SLEEP_PROCESSING(clock_mode_t powermode);
-extern void serial_wait_tx_complete(uint32_t uart_index);
-
+extern int serial_check_tx_ongoing();
```

Comment:
Should really be `bool`. I think only reason there's not much bool in the existing code is because it predates C99 being enabled in Mbed OS. Oh, and `(void)`. This isn't C++ - still need `(void)` in declarations, or else you're leaving parameters unspecified.

Reply:
I have changed this to bool.